unilog.go

```diff
@@ -224,31 +225,36 @@ func (u *Unilog) logLine(line string) {
 
 func (u *Unilog) run() {
 	for {
+		u.selector()
+	}
+}
+
+func (u *Unilog) selector() {
```

Comment:
Not loving this name... but can't think of something more descriptive at the moment. /: